### ISSUE TITLE
A way to set method properties

### ISSUE THREAD

**Issue by swlkr** [phel-lang/phel-lang#235]
I was working through 

https://github.com/walkor/Workerman#an-http-server

and there's currently no way to set a property on a method (without a setter).

Here's what I wanted to accomplish in phel:

```php
use Workerman\Worker;

$http_worker = new Worker('http://0.0.0.0:2345');
$http_worker->count = 4;
$http_worker->onMessage = function ($connection, $request) {
    $connection->send("Hello World");
};
```

I thought something like this might work:

```clojure
(def http-worker (php/new Worker "http://0.0.0.0:2345"))

; # maybe?
(php/-> worker count 4)

; # maybe this instead?
(php/= (php/-> worker count) 4)
```

**Comment by jenshaase** [comment]
I added this feature, released a new version (0.2.0) and updated the documentation on the website. You can now do this:
```
(php/oset (php/-> worker count) 4)
```

**Comment by swlkr** [comment]
Nice! This should fix my problems!
